### PR TITLE
Add coverage report to the GitHub Actions summary

### PR DESCRIPTION
**Pull request by zerolab**
See https://github.com/marksweb/django-nh3/actions/runs/7275054442

Screenshot 2023-12-20 at 12 05 24

### CODE REVIEW

**Comment by zerolab** [comment]
Note that with #19, the coverage report is more complete.

<details>
<summary>report</summary>

```
~ ❯❯❯ coverage report
Name                                      Stmts   Miss Branch BrPart  Cover   Missing
-------------------------------------------------------------------------------------
src/django_nh3/__init__.py                    0      0      0      0   100%
src/django_nh3/apps.py                        5      0      0      0   100%
src/django_nh3/forms.py                      18      0      4      0   100%
src/django_nh3/models.py                     29      3      6      0    86%   45-59
src/django_nh3/templatetags/__init__.py       0      0      0      0   100%
src/django_nh3/templatetags/nh3_tags.py      16      0      6      0   100%
src/django_nh3/utils.py                      19      0     10      0   100%
tests/__init__.py                             0      0      0      0   100%
tests/constants.py                            3      0      0      0   100%
tests/migrations/0001_initial.py              6      0      0      0   100%
tests/migrations/__init__.py                  0      0      0      0   100%
tests/settings.py                            15      0      0      0   100%
tests/test_forms.py                          13      0      4      0   100%
tests/test_models.py                         44      0      4      0   100%
tests/test_settings.py                       17      0      6      0   100%
tests/test_templatetags.py                   18      0      0      0   100%
tests/urls.py                                 3      3      0      0     0%   1-5
-------------------------------------------------------------------------------------
TOTAL                                       206      6     40      0    97%
```
</details>

**Comment by marksweb** [comment]
Should probably exclude the tests module from coverage. It's only worth tracking the test coverage of the app itself I think 🤔

**Comment by zerolab** [comment]
@marksweb the coverage docs recommend that tests are included and links to https://nedbatchelder.com/blog/202008/you_should_include_your_tests_in_coverage.html

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -33,8 +41,8 @@ jobs:
 
       - name: Install dependencies
         run: |
-          python -m pip install --upgrade pip setuptools wheel
-          python -m pip install --upgrade 'tox>=4.0.0rc3'
+          python -Im pip install --upgrade pip setuptools wheel
```
Comment (marksweb) [inline .github/workflows/main.yml:44]: Now there's another I'm not in the habbit of using.

**Review by marksweb — APPROVED** [review]
Love these improvements! Now I need to spread them around all my other projects!!

**Pull request merged**